README.md

```diff
@@ -26,9 +26,9 @@ Sign up to [get started for free](https://app.revenuecat.com/signup).
 
 *Purchases* and *RevenueCat* are clients for the [RevenueCat](https://www.revenuecat.com/) subscription and purchase tracking system.
 
-**Purchases** is the currently supported, production-ready, open source framework that provides a wrapper around `StoreKit` and the RevenueCat backend to make implementing in-app subscriptions in `Swift` or `Objective-C` easy - receipt validation and status tracking included! 
+**Purchases** is the v3 production-ready, open source framework that provides a wrapper around `StoreKit` and the RevenueCat backend to make implementing in-app subscriptions in `Swift` or `Objective-C` easy - receipt validation and status tracking included! 
```

Comment:
I think we can probably remove a lot of this now, like mentioning "production-ready" for v3 makes it seem like v4 isn't production-ready. 

And maybe we can actually remove the whole `Purchases` section now? It'll still be there for v3 tags, which I think is enough. 

What do you think?

Reply:
Yup that makes sense!

Reply:
How about this?

Reply:
❤️ it!